packages/commonwealth/client/scripts/views/components/edit_profile.tsx

```diff
@@ -169,8 +182,11 @@ const EditProfileComponent = () => {
   };
 
   useEffect(() => {
-    getProfile();
-  }, []);
+    if (!isFetchedOnMount.current) {
+      getProfile().catch(console.error);
+      isFetchedOnMount.current = true;
+    }
+  }, [getProfile]);
```

Comment:
instead of doing this, we should move it to react-query. However this PR is big so I do not insist doing it right now

Reply:
Created a ticket https://github.com/hicommonwealth/commonwealth/issues/7841